pkg/internal_registry/registry.go

```diff
@@ -0,0 +1,50 @@
+package registry
+
+import (
+	brokerModel "github.com/eclipse/che-plugin-broker/model"
+)
+
+func InternalRegistry() map[string]brokerModel.PluginMeta {
+
+	return map[string]brokerModel.PluginMeta{
+		"eclipse/cloud-shell/nightly": {
```

Comment:
Can we move it into yaml files... `/deploy/internal-registry/eclipse/cloud-shell/nightly`
Then it will be possible to customize it like with mount PVC or rebuild controller image with modified internal registries files?

Reply:
:+1: